includes/wikia/Extensions.php

```diff
@@ -0,0 +1,1892 @@
+<?php
+###############################################################################
+#  Overwrite some variables, load extensions, etc. Former CustomSettings.php  #
+###############################################################################
+
+# please use "date +%Y%m%d%H%M%S" as a proper value
+# don't update it unless you really need to
+$wgCacheEpoch = max( $wgCacheEpoch, '20080205154442' );
+
+###############################################################################
+# DC specific settings                                                        #
+###############################################################################
+switch ($wgWikiaDatacenter) {
+	case "res":
+		# PLATFORM-1740: disable task queue in Reston, it was calling SJC broker
+		$wgTaskBroker = false;
+		break;
+}
+
+/**
+ * define extension directory based on IP variable
+ */
+$wgExtensionsDirectory = $IP . "/extensions";
+
+/**
+ * $wgDefaultSkin is first set to 'oasis' in CommonSettings.php. Then the value is assigned to
+ * $wgDefaultSkinBeforeWF and $wgDefaultSkin is nullified. WikiFactor is supposed to assign
+ * some value to $wgDefaultSkin - a per-wiki setting. If it fails and at this point $wgDefaultSkin
+ * is still null, we use the cached value from $wgDefaultSkinBeforeWF.
+ */
+if ( is_null( $wgDefaultSkin ) ) {
+	$wgDefaultSkin = $wgDefaultSkinBeforeWF;
+}
+
+//can access ->cat_id or ->cat_name
+$wgHub = WikiFactory::getCategory($wgCityId);
+
+# Remove any skin(s) listed in UnSkipSkins from SkipSkins
+if( !empty($wgUnSkipSkins) && is_array($wgUnSkipSkins) ) {
```

Comment:
this is set on 3 wikis - imho let's set `wgSkipSkins` value for those and get rid of this